lib/commonAWS.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
```

Comment:
Definitely, we are just going to keep reusing this code for our AWS stuff.  Make a repo for it and add a readme with any specs you want and I'll switch over to it.